app/controllers/pages_controller.rb

```diff
@@ -35,8 +36,8 @@ def update
 
 private
 
-  def page_params(form_id)
-    params.require(:page).permit(:question_text, :question_short_name, :hint_text, :answer_type, :is_optional).merge(form_id:)
+  def page_params
+    params.require(:page).permit(:question_text, :question_short_name, :hint_text, :answer_type, :is_optional).merge(form_id: @form.id).merge(session:)
```

Comment:
Do we need to merge the whole session here or just add `session[:answer_type]`? I think the session has other stuff in it - e.g. csrf token etc.